types/react-router/test/EmptyObjectProps.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react'
+import {
+  BrowserRouter as Router,
+  Route
+} from 'react-router-dom'
+import { connect } from "react-redux"
+
+const EmptyPropsSfc = (props: {}) => (
+  <div>
+    <h2>Empty Props SFC</h2>
+  </div>
+)
+
+class EmptyPropsClassComponent extends React.Component<{}, void> {
+  render() {
+    return <div>
+      <h2>Empty Props Class</h2>
+    </div>
+  }
+}
+
+const ConnectedComponent = connect(() => ({}), {})((props) => {
+  return <div>
+    <h2>Empty Props Connected Class</h2>
+  </div>
+})
```

Comment:
I append `react-redux` connect test too.
But it's may redundant. Please advice.